src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayController.cs

```diff
@@ -217,6 +217,13 @@ public async Task<IActionResult> EvaluateTradingNamePage(TradingNamePageViewMode
             return RedirectToAction("ViewApplication", new { viewModel.ApplicationId });
         }
 
+        #region 1.3 - OrganisationStatus
+        [HttpGet("/Roatp/Gateway/{applicationId}/Page/1-30")]
+        public async Task<IActionResult> GetOrganisationStatus(Guid applicationId)
+        {
+            var username = _contextAccessor.HttpContext.User.UserDisplayName();
+            return View("~/Views/Roatp/Apply/Gateway/pages/OrganisationStatus.cshtml", await _mediator.Send(new GetOrganisationStatusRequest(applicationId, username)));
```

Comment:
Can we not let the framework work out the page rather than having the complete path each time?